libs/execution_context/ebpf_maps.c

```diff
@@ -705,6 +705,11 @@ _create_hash_map_internal(
         goto Done;
     }
 
+    retval = ebpf_safe_size_t_add(map_definition->value_size, supplemental_value_size, &actual_value_size);
```

Comment:
If LRU updates are in a 'hot' path, would we end up paying 'un-aligned access' performance penalties if the 'key-age' value location is not 64-bit aligned?  Maybe we could round up the ```map_definition->value_size``` to a 64-bit boundary, thereby ensuring that the 'key-age' location is 64-bit aligned as well.

Reply:
Fixed.